schema/project-level/project-schema.json

```
@@ -1998,6 +2008,76 @@
         }
       },
       "minProperties": 1
+    },
+    "Meeting": {
+      "title": "Meeting",
+      "description": "The details of a meeting.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "title": "ID",
+          "description": "A locally unique identifier for the meeting.",
+          "type": [
+            "string"
+          ]
+        },
+        "date": {
+          "title": "Date",
+          "description": "The date of the meeting.",
+          "type": [
+            "string"
+          ],
+          "format": "date-time"
+        },
+        "location": {
+          "title": "Location",
+          "description": "The location of the meeting.",
```

Update the name, title and description to match the referenced definition.

```suggestion
        "address": {
          "title": "Address",
          "description": "The address of the meeting.",
```